pcdsdevices/ui/BeckhoffAxisEPSCustom.embedded.ui

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>Form</class>
+ <widget class="QWidget" name="Form">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>886</width>
+    <height>179</height>
+   </rect>
+  </property>
+  <property name="sizePolicy">
+   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+    <horstretch>0</horstretch>
+    <verstretch>0</verstretch>
+   </sizepolicy>
+  </property>
+  <property name="minimumSize">
+   <size>
+    <width>0</width>
+    <height>0</height>
+   </size>
+  </property>
+  <property name="windowTitle">
+   <string>Form</string>
+  </property>
+  <property name="styleSheet">
+   <string notr="true"/>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <property name="spacing">
+    <number>3</number>
+   </property>
+   <property name="leftMargin">
+    <number>0</number>
+   </property>
+   <property name="topMargin">
+    <number>0</number>
+   </property>
+   <property name="rightMargin">
+    <number>0</number>
+   </property>
+   <property name="bottomMargin">
+    <number>0</number>
+   </property>
+   <item>
+    <widget class="TyphosPositionerRowWidget" name="TyphosPositionerWidget">
+     <property name="toolTip">
+      <string/>
+     </property>
+     <property name="readback_attribute" stdset="0">
+      <string>user_readback</string>
+     </property>
+     <property name="setpoint_attribute" stdset="0">
+      <string>user_setpoint</string>
+     </property>
+     <property name="low_limit_switch_attribute" stdset="0">
+      <string>low_limit_switch</string>
+     </property>
+     <property name="high_limit_switch_attribute" stdset="0">
+      <string>high_limit_switch</string>
+     </property>
+     <property name="low_limit_travel_attribute" stdset="0">
+      <string>low_limit_travel</string>
+     </property>
+     <property name="high_limit_travel_attribute" stdset="0">
+      <string>high_limit_travel</string>
+     </property>
+     <property name="velocity_attribute" stdset="0">
+      <string>velocity</string>
+     </property>
+     <property name="acceleration_attribute" stdset="0">
+      <string>acceleration</string>
+     </property>
+     <property name="error_message_attribute" stdset="0">
+      <string>plc.status</string>
+     </property>
+     <property name="show_expert_button" stdset="0">
+      <bool>true</bool>
+     </property>
+     <property name="alarmKindLevel" stdset="0">
+      <enum>TyphosPositionerWidget::HINTED</enum>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <layout class="QHBoxLayout" name="horizontalLayout">
+     <item>
+      <widget class="QLabel" name="label">
+       <property name="text">
+        <string>EPS:</string>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="PyDMShellCommand" name="PyDMShellCommand">
```

Comment:
This is implemented as a `PyDMShellCommand` widget that shells out to `pydm`.
Is this preferable to using a `PyDMRelatedDisplayButton` widget that opens a new window in the same process?